docs/_layouts/examples.html

```diff
@@ -12,6 +12,7 @@
     {% else %}
     <link rel="stylesheet" type="text/css" href="/build/css/build.css" />
     {% endif %}
+    <link rel="icon" href="https://assets.ubuntu.com/v1/ab36e6ed-vanilla_favicon_32px.png" type="image/x-icon" />
```

Comment:
Are we generally doing favicons as single size png?

Reply:
I remember there used to be more to it (https://css-tricks.com/favicon-quiz/) but maybe that's overkill / outdated.

Reply:
I just copied what we have in Vanilla docs.

Seems like it's not much different on vanillaframework.io site.

If we want to address it we should do it separately and update icons everywhere.